agent/azure-monitor-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/UrlParserPathTest.java

```diff
@@ -11,130 +11,80 @@ public class UrlParserPathTest {
 
   @Test
   public void testGetPathFromUrl() {
```

Comment:
url could be null or empty? If yes, it would be valuable to add tests for these cases.

Reply:
I added tests for empty and malformed. this method shouldn't be called with null url.